fava/application.py

```diff
@@ -290,6 +294,98 @@ def help_page(page_slug='_index'):
                            help_html=render_template_string(html))
 
 
+def _load_ingest_config(configpath):
```

Comment:
All this logic shouldn't be in `fava.application` but rather in `fava.core.import`